CMakeLists.txt

```diff
@@ -77,7 +82,7 @@ ament_export_include_directories(include)
 ament_export_libraries(${PROJECT_NAME})
 ament_export_dependencies(${dependencies})
 
-pluginlib_export_plugin_description_file(nav2_core fcpp_plugin.xml)
+pluginlib_export_plugin_description_file(nav2_core global_planner_plugin.xml)
```

Comment:
`global_planner` might not be the best name? Kind of generic.

Reply:
I started with an example, you are right, just changed it